### ISSUE TITLE
Issue using object input

### ISSUE THREAD

**Issue by Brunomachadob** [ClickerMonkey/unitz-ts#5]
Hello!

Am I doing something wrong here? I as understood from the docs, I could use an object input to do a unit conversion.

```js
const inLb: number = uz({ value: 200, unit: "kg" }).convert("lb").value
//inLib = 0;
```

It seems to work when using

```js
const inLb: number = uz("200kg").convert("lb").value
//inLib = 440.925;
```

Here I have a running example: 
[Example](https://stackblitz.com/edit/typescript-vp4n9m)

Thanks!

**Comment by ClickerMonkey** [comment]
Looking into it

**Comment by ClickerMonkey** [comment]
Fix committed in version `1.0.1`

**Comment by Brunomachadob** [comment]
That was fast!
Thanks!
